app/src/main/java/org/oppia/app/home/WelcomeViewModel.kt

```diff
@@ -4,9 +4,8 @@ import androidx.databinding.ObservableField
 import androidx.lifecycle.ViewModel
 
 /** [ViewModel] for user app usage history. */
-class UserAppHistoryViewModel : HomeItemViewModel() {
+class WelcomeViewModel : HomeItemViewModel() {
   var isAppAlreadyOpened = ObservableField<Boolean>(false)
```

Comment:
This can be removed too. As it won't be getting used anymore.

Reply:
Done :)